src/condor_ce_info_status

```diff
@@ -14,6 +14,17 @@ import htcondor
 
 import htcondorce.info_query
 
+# Add htcondor.param.get() to old versions of condor-python that don't have it
+try:
+    htcondor.param.get
```

Comment:
When was `param.get` introduced - and what's the minimum version of HTCondor supported by the CE?

I suspect we can simply drop the workaround.